README.md

```diff
@@ -38,7 +38,7 @@ If you have not added `$GOPATH/bin` to your `$PATH`, then you may
 alternatively run `./mage build` and manually move the `bin/ocm-addons`
 binary to any other directory in your `$PATH`.
 
-### Option 3: Install using 'install.sh' (Not supported for Windows)
+### Option 3: Install using 'install.sh' (Not supported for Windows or Mac)
```

Comment:
Can you paste the output from a failed run of the `install.sh` script on MacOS?

I'm assuming the arch type is not properly mapped for Apple Silicon, but if given the output I'll ask one of the Mac users on our team to work on a fix.

Reply:
tar: Error opening archive: Unrecognized archive format